docs/en/tasks/2d_animal_keypoint.md

```diff
@@ -35,7 +35,7 @@ MMPose supported datasets:
 
 For [Animal-Pose](https://sites.google.com/view/animal-pose/) dataset, we prepare the dataset as follows:
 
-1. Download the images of [PASCAL VOC2012](http://host.robots.ox.ac.uk/pascal/VOC/voc2012/#data), especially the five categories (dog, cat, sheep, cow, horse), which we use as trainval dataset.
+1. Download the images of [PASCAL VOC2012](http://host.robots.ox.ac.uk/pascal/VOC/voc2012/VOCtrainval_11-May-2012.tar), especially the five categories (dog, cat, sheep, cow, horse), which we use as trainval dataset.
```

Comment:
```suggestion
1. Download the images of [PASCAL VOC2012](http://host.robots.ox.ac.uk/pascal/VOC/voc2012/VOCtrainval_11-May-2012.tar), especially the five categories (dog, cat, sheep, cow, horse), which we use as the trainval dataset. See [PASCAL VOC 2012 official site](http://host.robots.ox.ac.uk/pascal/VOC/voc2012) for more information.
```

Reply:
I suggest also providing the link to the web page for users who want to learn more about the dataset.